README.md

```diff
@@ -32,66 +32,91 @@ A tutorial for manim, a mathematical animation engine made by 3b1b for Python.
 * [Resources](#Resources)
 * [Further Work](#Further-Work)
 * [Acknowledgements](#Acknowledgements)
+
 ## Installations
-This installation and guide is meant for linux users. We will start by installing prerequisites. 
 
-Lets first get the repo for manim, using:
-``` bash
-    git clone https://github.com/3b1b/manim.git
-```
+Assuming you have Python 3 installed and set up properly on your machine.
+
+Since [a commit][commit] by [@JohnAZoidberg][committer] in April, manim has been made installable using Python's tool `pip`, so the installation process is much simpler than before.
 
+[commit]: https://github.com/3b1b/manim/commit/2a6918cc30336544856a68f065a21ba2b769b01c
+[committer]: https://github.com/JohnAZoidberg
+
+### Linux
 We will start with installing some system requirements: Cairo, Latex, ffmpeg and sox. 
 
-Install by running the following commands on the terminal: 
+```shell
+$ sudo apt-get install libcairo2-dev libjpeg-dev libgif-dev 
+$ sudo apt-get install texlive-latex-base texlive-full texlive-fonts-extra
+```
 
-``` bash
-    sudo apt install SystemReq
+After having these libraries and tools installed in your computer, you can now type this command to finish the last step of the installation
```

Comment:
Test that this works. As far as I know, this doesn't work yet on Linux or Windows.

Reply:
Keep original. Only add mac additions